historical_system_profiles/db_interface.py

```diff
@@ -37,6 +39,7 @@ def get_hsps_by_inventory_id(inventory_id, account_number):
         HistoricalSystemProfile.account == account_number,
         HistoricalSystemProfile.inventory_id == inventory_id,
     )
+    ensure_correct_system_count(inventory_id, query)
```

Comment:
Is this a string or an array of string? It may not work as expected if it's just a string:

```python
>>> x = "57e7b37c-9947-11ea-94d4-98fa9b07d419"
>>> len(x)
36
```
```python
>>> y = ["57e7b37c-9947-11ea-94d4-98fa9b07d419"]
>>> len(y)
1
```

Reply:
Good catch, as this one is only a single ID and not a list as the others have been.